admin_interface/templates/admin/base_site.html

```diff
@@ -67,14 +67,18 @@
 {% get_admin_interface_theme as theme %}
 <h1 id="site-name">
     {% if theme.logo_visible %}
-        {% if theme.logo %}
-        <img class="logo" style="display:none;" src="{{ theme.logo.url }}" {% if theme.logo.width %}width="{{ theme.logo.width }}"{% endif %} {% if theme.logo.height %}height="{{ theme.logo.height }}"{% endif %}>
-        {% else %}
-        <img class="logo default" style="display:none;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" width="104" height="36">
-        {% endif %}
+        <a href="{% url 'admin:index' %}">
+            {% if theme.logo %}
+            <img class="logo" style="display:none;" src="{{ theme.logo.url }}" {% if theme.logo.width %}width="{{ theme.logo.width }}"{% endif %} {% if theme.logo.height %}height="{{ theme.logo.height }}"{% endif %}>
+            {% else %}
+            <img class="logo default" style="display:none;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" width="104" height="36">
+            {% endif %}
+        </a>
     {% endif %}
     {% if theme.title_visible %}
-    <span>{% if theme.title %}{% trans theme.title %}{% else %}{{ site_header|default:_('Django administration') }}{% endif %}</span>
+    <a href="{% url 'admin:index' %}">
+        <span>{% if theme.title %}{% trans theme.title %}{% else %}{{ site_header|default:_('Django administration') }}{% endif %}</span>
+    </a>
```

Comment:
Why two `<a>` elements? 
I think that just one `<a>` element would be enough to wrap both the logo and the title in case they are both visible.